pyaedt/modules/PostProcessor.py

```diff
@@ -2539,7 +2533,7 @@ def export_field_file_on_grid(
         grid_step : list, optional
             The ``[x, y, z]`` coordinates for the step size of the grid.
             The default is ``[0, 0, 0]``.
-        isvector : bool, optional
+        is_vector : bool, optional
             Whether the quantity is a vector. The  default is ``False``.
         intrinsics : str, optional
             This parameter is mandatory for a frequency field
```

Comment:
Same comment here. It needs a description and a concluding period on the existing sentence.